projects/packages/my-jetpack/_inc/components/product-detail-card/style.module.scss

```diff
@@ -99,13 +99,6 @@
 		height: auto;
 		line-height: 26px;
 		align-self: flex-end;
-
-		&.is-bundle {
-			&:hover {
-				background-color: var(--product-card-action-button-bg-color-hover);
-				color: var(--product-card-action-button-color-hover);
-			}
-		}
```

Comment:
Why do you remove these lines? It allows switching the color and background of the button when it's bundle type.

before | after
----|----
<img width="775" alt="image" src="https://user-images.githubusercontent.com/77539/154361462-578fc7eb-13ac-4463-aa8f-c05b8ce05890.png"> | <img width="775" alt="image" src="https://user-images.githubusercontent.com/77539/154361394-1a6cbee2-2def-4ceb-9941-3eb4f1365e97.png">

Reply:
fixed.